openslides/users/static/templates/users/user-list.html

```diff
@@ -134,94 +105,280 @@ <h1 translate>Participants</h1>
-              ng-class="reverse ? 'fa-sort-desc' : 'fa-sort-asc'">
-          </i>
+    <!-- main table -->
+    <div class="row data-row" ng-mouseover="user.hover=true"
+      ng-mouseleave="user.hover=false"
+      ng-class="{'projected': user.isProjected()}"
+      ng-repeat="user in usersFiltered
+      | limitTo : itemsPerPage : limitBegin">
 
-        <!-- present column -->
-        <th os-perms="users.can_see_extra_data" ng-click="toggleSort('is_present')" class="sortable minimum">
-          <translate>Present</translate>
-          <i class="pull-right fa" ng-show="sortColumn === 'is_present' && header.sortable != false"
-              ng-class="reverse ? 'fa-sort-desc' : 'fa-sort-asc'">
-          </i>
-    <tbody>
-      <tr ng-repeat="user in usersFiltered | limitTo : itemsPerPage : limitBegin" class="animate-item"
-          ng-class="{ 'activeline': user.isProjected(), 'selected': user.selected  }">
-        <!-- projector column -->
-        <td ng-show="!isSelectMode" os-perms="core.can_manage_projector">
-          <projector-button model="user" default-projector-id="defaultProjectorId">
-          </projector-button>
-        <!-- selection column -->
-        <td ng-show="isSelectMode" os-perms="users.can_manage" class="deleteColumn">
-          <input type="checkbox" ng-model="user.selected">
-        <!-- user data colums -->
-        <td ng-mouseover="user.hover=true" ng-mouseleave="user.hover=false">
-          <strong>
-            <i ng-show="user.is_active === false" class="fa fa-ban"
-              title="{{ 'Is inactive' | translate }}"></i>
-            <i ng-show="user.is_committee" class="fa fa-users"
-              title="{{ 'Is a committee' | translate }}"></i>
-            <a ui-sref="users.user.detail({id: user.id})">{{ user.get_short_name() }}</a>
-          </strong>
-          <div ng-if="user.comment">
-            <small><i class="fa fa-info-circle"></i> {{ user.comment }}</small>
+      <!-- select column -->
+      <div ng-show="isSelectMode" os-perms="users.can_manage" class="col-xs-1 centered">
+        <i class="fa text-danger pointer" ng-click="user.selected=!user.selected"
+        ng-class="user.selected ? 'fa-check-square-o' : 'fa-square-o'"></i>
+      </div>
+      <!-- projector column -->
+      <div class="col-xs-1 centered projector" os-perms="core.can_manage_projector">
+        <projector-button model="user", default-projector-id="defaultProjectorId">
+        </projector-button>
+      </div>
+      <!-- main content column -->
+      <div class="col-xs-6 content">
+        <div class="spacer-right"> <!-- horizontal block -->
+            <strong>
+              <i ng-style="{'visibility': user.is_active === false ? 'visible' : 'hidden'}" class="fa fa-ban"
+                title="{{ 'Is inactive' | translate }}"></i>
+              <i ng-style="{'visibility': user.is_committee ? 'visible' : 'hidden'}" class="fa fa-users"
```

Comment:
Suggestion: Use 'fa-university' icon for a better distinction of user group icon.